src/main/java/org/opensrp/web/controller/MultimediaController.java

```diff
@@ -259,4 +278,13 @@ private void writeFileNotFound(HttpServletResponse response) throws IOException
 		outputStream.write(errorMessage.getBytes(Charset.forName("UTF-8")));
 		outputStream.close();
 	}
+
+	private void fileWithSpecialCharactersError(HttpServletResponse response) throws IOException {
+		String errorMessage = "Sorry. File Name should not contain any special character";
+		logger.info(errorMessage);
```

Comment:
shouldn't be an error or warning and not an info

Reply:
Done